packages/grid/_modules_/grid/components/ScrollArea.tsx

```diff
@@ -2,75 +2,79 @@ import * as React from 'react';
 import { COL_REORDER_START, COL_REORDER_STOP, SCROLLING } from '../constants/eventsConstants';
 import { ScrollParams, useApiEventHandler } from '../hooks';
 import { ApiRef } from '../models';
+import { classnames } from '../utils';
 import { ApiContext } from './api-context';
 
-enum Direction {
-  Left = 'left',
-  Right = 'right',
-}
-
-type ScrollDirection = 'left' | 'right';
-
 interface ScrollAreaProps {
-  scrollDirection: ScrollDirection;
+  scrollDirection: 'left' | 'right';
 }
-export const ScrollArea: React.FC<ScrollAreaProps> = React.memo(
-  ({ scrollDirection }: ScrollAreaProps) => {
-    const api = React.useContext(ApiContext);
-    const cursorPossitionX = React.useRef<number | null>(null);
-    const cssClass = `MuiDataGrid-scrollArea ${
```

Comment:
we use `classename` to (and will move to clsx) to build class names

Reply:
Ok, thanks I'll keep that in mind